pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/TopicOwnershipListener.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.namespace;
+
+import org.apache.pulsar.common.naming.TopicName;
+
+import java.util.function.Predicate;
+
+/**
+ * Listener for <code>Topic</code> ownership changes
+ */
+public interface TopicOwnershipListener {
```

Comment:
Actually after thinking a bit more, I am not sure we need a topic ownership listener right now. 

- for TC, when a namespace bundle is loaded, we will know the partitions of the TC topic belong to that bundle and load TC accordingly.
- for KoP, it will do the same as TC.
- for namespace change streams, we just need the bundle information.

Hence, I think we can just add the namespace bundle listener for now. if we need a topic listener, we can add it in future.

Reply:
ok, i will remove the topic ownership listener